ejs-views/partials/search_description.ejs

```diff
@@ -0,0 +1,12 @@
+<%
+  let serviceSpecified = locals.serviceType && locals.serviceName;
+  let serviceTypeVerb = locals.serviceType === 'provided' ? 'provide' : 'consume';
+%>
+
+<% if (serviceSpecified)  { %>
+<div class="text-center mt-4 mb-4">
+  <p>
+    Displaying all packages which <%= serviceTypeVerb %> the <strong><%= locals.serviceName %></strong> service.
+  </p>
+</div>
+<% } %>
```

Comment:
This is currently included on just `/packages` because these parameters don't have any effect on `/packages/search`. When we combine those two endpoints into one, that'll change.

Reply:
I'll do my best to keep this in mind once that's fixed